### PR TITLE
Update some of the 3D particles pages for 4.2

### PR DESCRIPTION
**Pull request by skyace65**
Updates the particles pages with information primarily from [this blogpost](https://godotengine.org/article/progress-report-state-of-particles/) and the class reference page. This isn't everything from 4.2, most of the new process materials properties are still not documented however I've re-organized the page so everything is in the same order as what's in the 4.2 editor, updating it in the future should be easier now. Here is a list of all changes:

- Mentions the new limitations of CPU particles
- Mentions that you can now convert from CPU particles to GPU particles
- Documents the new GPUParticles nodes properties
- Updates 2 images on the GPUParticles node page
- Documents the new damping as friction property
- Re-orders the process materials page

I have no plans to add info for the other new properties in this PR, it can be merged as is after review.

### CODE REVIEW

File: tutorials/3d/particles/properties.rst [hunk 1]
```diff
@@ -16,6 +16,12 @@ want to activate or deactivate particle systems dynamically.
 The ``Amount`` property controls the maximum number of particles visible at any given time. Increase the
 value to spawn more particles at the cost of performance.
 
+The ``Amount Ratio`` property is the percentage of particles compared to the ammount that will be emitted.
+If it's less than ``1.0`` the ammount of particles emitted through the lifetime will be the ``Ammount`` *
+``Amount Ratio``. Changing this value while emitted doesn't affect already created particles and doesn't
+cause the particle system to restart. Its useful for making effects where the number of emitted particels
```
Comment (AThousandShips) [inline tutorials/3d/particles/properties.rst:22]:
```suggestion
cause the particle system to restart. It's useful for making effects where the number of emitted particels
```

File: tutorials/3d/particles/properties.rst [hunk 2]
```diff
@@ -16,6 +16,12 @@ want to activate or deactivate particle systems dynamically.
 The ``Amount`` property controls the maximum number of particles visible at any given time. Increase the
 value to spawn more particles at the cost of performance.
 
+The ``Amount Ratio`` property is the percentage of particles compared to the ammount that will be emitted.
```
Comment (AThousandShips) [inline tutorials/3d/particles/properties.rst:19]: "percentage" might be misleading as the value is `[0,1]` not `[0,100]` AFAIK

**Comment by skyace65** [comment]
Fixed.

**Review by mhilbrunner — APPROVED** [review]

**Comment by mhilbrunner** [comment]
Thanks! Merged.

**Pull request merged**